workspaces/global-floating-action-button/plugins/global-floating-action-button/src/plugin.ts

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright 2025 The Backstage Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import {
+  createComponentExtension,
+  createPlugin,
+} from '@backstage/core-plugin-api';
+
+export const globalFloatingActionButtonPlugin = createPlugin({
+  id: 'global-floating-action-button',
+});
+
+/**
+ * @public
+ * Floating Button Plugin
+ */
+export const GlobalFloatingButton = globalFloatingActionButtonPlugin.provide(
+  createComponentExtension({
+    name: 'GlobalFloatingButton',
+    component: {
+      lazy: () => import('./components').then(m => m.FloatingButtons),
+    },
+  }),
```

Comment:
I think you should eighter rename the export or the file here. So that both matches. This makes it much easier in the future to understand the plugin.

Also, please use the full path of that component in the import('...') statement so allow the build process build smaller build artifacts / chunks.